src/plugins/graph/components/legend/layer-legend.tsx

```diff
@@ -129,7 +137,7 @@ const SingleLayerLegend = observer(function SingleLayerLegend(props: ILegendPart
         return isSharedDataSet(sharedModel) && sharedModel.dataSet.id === datasetId;
       });
       const foundProviderId = (foundSharedModel as SharedDataSetType)?.providerId;
-      const foundTile = smm.getSharedModelTiles(foundSharedModel)?.find(tile => tile.id === foundProviderId);
+      const foundTile = smm.getSharedModelTiles(foundSharedModel)?.find(_tile => _tile.id === foundProviderId);
```

Comment:
Changed this because I had to deconstruct `tile` above in order to have the ITileModel to log